src/test/rgw/rgw_multi/zone_ps.py

```diff
@@ -1,14 +1,14 @@
 import logging
-import httplib
 import ssl
 import urllib
-import urlparse
 import hmac
 import hashlib
 import base64
 import xmltodict
+from six.moves import http_client
+from six.moves.urllib.parse import urlparse
 from time import gmtime, strftime
-from .multisite import Zone
+from tasks.rgw_multi.multisite import Zone
```

Comment:
this breaks `src/test/rgw/test_multi.py` script that runs this code outside of teuthology:
```
    from tasks.rgw_multi.conn import get_gateway_connection, get_gateway_secure_connection
ImportError: No module named tasks.rgw_multi.conn
```

Reply:
where the failure can been observed?

Reply:
> ~/ceph/build $ nosetests ../src/test/rgw/test_multi.py

Reply:
This is definitely incorrect way of running tests, can this command be prepended with PYTHONPATH or something?

Reply:
is it in run-make-check?

Reply:
no, its a standalone script to run the multisite test cases under src/test/rgw/rgw_multi/. it imports things from rgw_multi.*

for teuthology, there's a symlink to this directory at qa/tasks/rgw_multi used by qa/tasks/rgw_multisite.py and rgw_multisite_tests.py

because of that symlink, there is no single absolute path for those imports, which is why this broke. does python3 really not support relative imports like `from .multisite import Zone`?

Reply:
ah... I see, it is even not a part of teuthology

Reply:
@cbodley i am very sorry for the regression! will review your fix of #34190